docs/source/dataframe-design.rst

```diff
@@ -69,6 +69,56 @@ This keyword is available on all functions/methods that take user provided
 callables (e.g. ``DataFrame.map_partitions``, ``DataFrame.apply``, etc...), as
 well as many creation functions (e.g. ``dd.from_delayed``).
 
+Categoricals
+------------
+
+Dask dataframe divides `categorical data`_ into two types:
+
+- Known categoricals have the ``categories`` known statically (on the ``_meta``
+  attribute). Each partition **must** have the same categories as found on the
+  ``_meta`` attribute.
+- Unknown categoricals don't known the categories statically, and may have
+  different categories in each partition.  Internally, unknown categoricals are
+  indicated by the presence of ``dd.utils.UNKNOWN_CATEGORIES`` in the
+  categories on the ``_meta`` attribute.  Since most dataframe operations
```

Comment:
What if there are other categories in there as well?  Or is the condition that there must be a single element with value `UNKNOWN_CATEGORIES`?